specification/appendix-d-observability.md

```diff
@@ -19,49 +19,55 @@ This is particularly relevant to telemetry-related [hooks](./sections/04-hooks.m
 
 ### Provider Metadata
 
-| Log Record Attribute         | Provider Metadata Field | Requirement level | Notes                                                                                            |
-| ---------------------------- | ----------------------- | ----------------- | ------------------------------------------------------------------------------------------------ |
-| `feature_flag.provider_name` | `name`                  | `Recommended`     | The name of the provider as defined in the `provider metadata`, available in the `hook context`. |
+| Log Record Attribute         | Provider Metadata Field | Requirement level | Type     | Notes                                                                                            |
+| ---------------------------- | ----------------------- | ----------------- | -------- | ------------------------------------------------------------------------------------------------ |
+| `feature_flag.provider.name` | `name`                  | `Recommended`     | `string` | The name of the provider as defined in the `provider metadata`, available in the `hook context`. |
+
+## History
+
+Feature flags in the OpenTelemetry semantic conventions are currently in development and are marked as experimental.
+The following table describes the history of changes to the OpenTelemetry feature flag log records as it progresses towards a stable release.
+
+| Original Field Name                     | New Field Name                | Semantic Convention Release                                                            |
+| --------------------------------------- | ----------------------------- | -------------------------------------------------------------------------------------- |
+| `feature_flag.evaluation.error.message` | `error.message`               | [v1.32.0](https://github.com/open-telemetry/semantic-conventions/releases/tag/v1.32.0) |
+| `feature_flag.variant`                  | `feature_flag.result.variant` | [v1.32.0](https://github.com/open-telemetry/semantic-conventions/releases/tag/v1.32.0) |
+| `feature_flag.evaluation.reason`        | `feature_flag.result.reason`  | [v1.32.0](https://github.com/open-telemetry/semantic-conventions/releases/tag/v1.32.0) |
+| `value`                                 | `feature_flag.result.value`   | Unreleased                                                                             |
+| `feature_flag.provider_name`            | `feature_flag.provider.name`  | Unreleased                                                                             |
 
 ## Footnotes
 
-[^1]: Include if and only if an error occurred during a flag evaluation.
-[^2]: The `variant` field is required if the `value` field is not set.
-[^3]: The `value` field is required if the `variant` field is not set.
+[^1]: The `variant` field is required if the `value` field is not set.
+[^2]: The `value` field is required if the `variant` field is not set.
```

Comment:
This is technically correct, but I think it doesn't communicate clearly when you would use one or the other. In openfeature you almost always want `variant` right? `value` is only there as a fallback for when `variant` doesn't exist or isn't meaningful.

Reply:
I've updated this section. Could you please re-review?